tests/test_updater_top_level_update.py

```diff
@@ -434,6 +434,18 @@ def test_new_targets_expired(self) -> None:
 
         self._assert_files_exist(["root", "timestamp", "snapshot"])
 
+    def test_compute_metafile_hashes_length(self) -> None:
+        self.sim.compute_metafile_hashes_length = True
+        self.sim.update_snapshot()
+        self._run_refresh()
+
+        self.sim.compute_metafile_hashes_length = False
+        self.sim.update_snapshot()
+        self._run_refresh()
+
+        self.assertIsNone(self.sim.timestamp.snapshot_meta.hashes)
+        self.assertIsNone(self.sim.timestamp.snapshot_meta.length)
```

Comment:
If we assert about `sim.timestamp.snapshot_meta`, should we do it for `sim.snapshot.meta["targets.json"]` too?
Or if we consider it a regression test for this particular bug should we add 

```
self.assertIsNotNone(self.sim.timestamp.snapshot_meta.hashes)
self.assertIsNotNone(self.sim.timestamp.snapshot_meta.length)
```
after we switch  self.sim.compute_metafile_hashes_length to True?

Reply:
I don't think we necessarily need to assert anything about simulator -- I get that this was fixing a bug in the simulator but the simulator still just a test utility and we know that this updater test happens to regression test that bug as well.

Sure, we could write a test suite for simulator itself but I don't think think it's necessary

Reply:
@ivanayov I guess I agree with Jussi. Can you remove the two asserts above and I think we'll be all set.

Reply:
@sechkova just addressed your comment